sdb/commands/zfs/vdev.py

```diff
@@ -68,6 +69,29 @@ def __init__(self,
         if self.args.weight:
             self.arg_list.append("-w")
 
+    #
+    # Iterate over the metaslabs to accumulate histogram data.
+    #
+    def sum_histograms(self,
+            metaslabs: Iterable[drgn.Object]):
+        shift = -1;
+        first_time = True
+        histsum: List[int] = [];
+        for msp in metaslabs:
+            if msp.ms_sm == sdb.get_typed_null(msp.ms_sm.type_):
```

Comment:
It looks like this is the canonical way of doing this, but it seems pretty tricky just to check for `null`.  As an orthogonal cleanup, maybe we should change this helper function.  It looks like all the callers of `get_typed_null(type)` could be changed to a simpler-to-use helper, something like:
```python
def is_null(obj: drgn.Object) -> bool:
  return obj == drgn.NULL(prog, obj.type_)
```

Reply:
Yup, I will file a new bug for this.

Reply:
Filed: https://github.com/delphix/sdb/issues/258